src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -176,7 +177,7 @@ const AuditorRunTestForm: React.FC<IAuditorRunTestForm> = ({
             name: name,
             version: version,
             subject: subject,
-            githubToken: accessToken || null,
+            githubToken: accessToken,
```

Comment:
:) FYI - This is needed, as per the API githubToken can be a valid value or null if empty string. I've added this back @EehMauro